coderd/database/queries/users.sql

```diff
@@ -266,4 +275,3 @@ RETURNING id, email, last_seen_at;
 -- AllUserIDs returns all UserIDs regardless of user status or deletion.
 -- name: AllUserIDs :many
 SELECT DISTINCT id FROM USERS;
-
```

Comment:
Do you have `editorconfig` installed? Should help in avoiding the trailing newline being trimmed.

Reply:
it's getting trimmed because this file currently ends with two newlines, so it still ends with one. I don't use editorconfig, but I have VSCode configured to always make sure files end with a newline.